includes/api/ApiQueryLogEvents.php

```diff
@@ -157,13 +154,33 @@ public function execute() {
 		$count = 0;
 		$res = $this->select( __METHOD__ );
 		$result = $this->getResult();
+
+		// SUS-2779
+		$userIds = $this->getFieldFromResults( $res, 'log_user' );
+		$userNames = $this->getIndexedUserNames( $userIds );
+
 		foreach ( $res as $row ) {
 			if ( ++ $count > $limit ) {
 				// We've reached the one extra which shows that there are additional pages to be had. Stop here...
 				$this->setContinueEnumParameter( 'start', wfTimestamp( TS_ISO_8601, $row->log_timestamp ) );
 				break;
 			}
 
+			// SUS-2779
+			if ( $this->fld_userid ) {
+				$row->user_id = $row->log_user;
+			}
+
+			if ( $this->fld_user ) {
+				if ( isset( $userNames[$row->log_user] ) ) {
+					$row->user_name = $userNames[$row->log_user];
+				} else {
+					$row->user_name = 'unknown';
+					\Wikia\Logger\WikiaLogger::instance()
+						->warning( "User with id {$$row->log_user} was not found" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the use of this variable variable "$$row". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1599)